test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -12284,13 +12323,14 @@ def forward(self, x):
 
     @skipIfUnsupportedMinOpsetVersion(10)
     def test_qat_avg_pool2d(self):
-        model = torch.nn.Sequential()
-        model.add_module("quant", torch.quantization.QuantStub())
-        model.add_module(
-            "avgpool", torch.nn.AvgPool2d(kernel_size=3, stride=2, padding=1)
+        layers = OrderedDict(
```

Comment:
I'd in-line this into the constructor call rather than assign to a local. The constructor only takes one argument so it's plenty readable.

Also I don't think naming the modules is really helpful. Can we pass in a tuple or list of modules without names?

Reply:
> Also I don't think naming the modules is really helpful. Can we pass in a tuple or list of modules without names?

Good catch. I was working on a parameterized version of the test which needed the module names to fuse/manipulate the modules. This is the remnant of it. I will remove.

> I'd in-line this into the constructor call rather than assign to a local. The constructor only takes one argument so it's plenty readable.

Sure, initially done to remove a level of indent since the model initializations got long

Reply:
Done